manifests/apache.pp

```diff
@@ -55,8 +55,11 @@
     ],
     'request_headers' => [
       "unset ${remote_user_environ_header}",
-      "set ${remote_user_environ_header} \"%{SSL_CLIENT_S_DN_CN}s\" env=SSL_CLIENT_S_DN_CN",
     ],
+    'custom_fragment' => epp('pulpcore/apache-if.epp', {
```

Comment:
I had to switch to custom_fragment as that is what puppet-postgresql provides and thus I added it to https://github.com/theforeman/puppet-pulpcore/pull/186/files#diff-6c47f87fb2ecb452123d65c6aa6c6d16bf34eeb418e70b2a6fe2e04efff890caR55 

The resulting formatting of the Apache configuration is ugly no matter what I did. The current way I have it looked the best but makes the templates kinda ugly.